app/[locale]/login/loginpage.tsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import React from 'react';
+import { FcGoogle } from 'react-icons/fc';
+
+import { getTranslations } from '~/i18n/translations';
+
+// Define an interface for the text object
+
+const LoginPage = async ({ locale }: { locale: string }) => {
+  const text = (await getTranslations(locale)).LoginText;
+  return (
+    <div className="bg-loginBackground flex h-auto grow flex-col items-center justify-center rounded-lg bg-background bg-contain bg-center bg-no-repeat bg-blend-overlay ">
+      <div className="mx-16 flex h-auto w-full grow flex-col items-center justify-center">
+        <header className="mb-8 text-center">
+          <h3 className="mt-4 text-primary-700">{text.title}</h3>
+        </header>
+        <div className="w-full max-w-lg space-y-6">
+          <div className="flex flex-col space-y-2">
+            <h6 className="text-primary-700">{text.enterEmail}</h6>
+            <input
+              id="email"
+              placeholder={text.enterEmail}
+              type="email"
+              className="outline-red-100 rounded-md border border-primary-500 p-5 outline-offset-2 ring-0"
+            />
+          </div>
+          <button className="text-md w-full rounded-md bg-primary-700 p-4 text-neutral-50 hover:bg-primary-900">
+            {text.continueButton}
+          </button>
+          <div className="flex items-center justify-center">
+            <div className="h-px w-full bg-primary-700" />
+            <span className="mx-2 text-sm text-primary-700">{text.orText}</span>
+            <div className="h-px w-full bg-primary-700" />
+          </div>
+          <Link href="/api/auth/signin">
```

Comment:
This is wrong, use the JS function for sign in instead of linking to the API route. See NKSS' code:
https://github.com/nkss-dev/node-modules/blob/main/app/breadcrumb.tsx#L207

This has a caveat of the button being client side, but the problem with just linking to the route is that it doesn't directly show the Google sign in modal, you go through a bootleg modal first adding an unnecessary click + bad UI